Revision Tests/maxi_stepanov.md

```diff
@@ -0,0 +1,32 @@
+## Please translate into English
+
+**_1. Если бы я был умнее (по жизни) то я бы сделал вчера бэкап всей базы и только потом применял свой ПР к коду на проде. Жаль что я не умный как мой папа._**
+
+> If I’d had smart, I would have done the backup yesterday and the apply my PR to the production.  I wish I was smart like my father.
+
+**_2. Если бы все студенты читали теорию и только потом писали перевод анализируя предложения, то прогресс бы был очевиден. Жаль что все хотят выучить английский брутфорсом, минуя ранние этапы обучения и анализа и отработки навыков до автоматизма. Я бы хотел чтобы вы не торопились._**
+
+> If all students read the theory and after that write the translation of the sentence with the analysis, then progress would be obvious. I wish everyone didn’t learn English with the brute force, avoiding the initial steps of education and analysis, and mastering the skills until it becomes second nature.  I wish you didn’t hurry up.
```

Comment:
after that wrote the translation analyzing the sentence
and drilling down the skills
I would like you not to hurry up